aws-event-sources-operator/bundle/0.0.1/aws-event-sources-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "sources.triggermesh.com/v1alpha1",
+          "kind": "AwsEventSources",
+          "metadata": {
+            "name": "example-awseventsources"
+          },
+          "spec": {
+            "adapters": {
+              "awscodecommit": {
+                "image": "gcr.io/triggermesh/awscodecommitsource:v0.3.0"
+              },
+              "awscognitoidentity": {
+                "image": "gcr.io/triggermesh/awscognitoidentitysource:v0.3.0"
+              },
+              "awscognitouserpool": {
+                "image": "gcr.io/triggermesh/awscognitouserpoolsource:v0.3.0"
+              },
+              "awsdynamodb": {
+                "image": "gcr.io/triggermesh/awsdynamodbsource:v0.3.0"
+              },
+              "awskinesis": {
+                "image": "gcr.io/triggermesh/awskinesissource:v0.3.0"
+              },
+              "awssns": {
+                "image": "gcr.io/triggermesh/awssnssource:latest"
```

Comment:
What about SNS? It was also part of the v0.3.0 release.

Reply:
good catch!